e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-save-and-back.spec.js

```diff
@@ -21,6 +23,17 @@ const {
   },
 } = FIELD_IDS.INSURANCE;
 
+const { taskList } = partials.insurancePartials;
+const task = taskList.prepareApplication.tasks.exporterBusiness;
+
+const { IN_PROGRESS } = TASKS.STATUS;
+
+const taskInProgress = (status) => {
+  task.status().invoke('text').then((text) => {
+    expect(text.trim()).equal(status);
+  });
+};
```

Comment:
Not a blocker for this PR but we could move this to a cypress command so in each test we can just do `cy.checkTaskStatus()` and pass the status in 

```js
const checkTaskStatus = (status) => { ...
```